.circleci/config.yml

```diff
@@ -138,7 +141,7 @@ run_mypy: &run_mypy
    - run:
        name: Run type-checking (mypy)
        command: |
-         mypy --ignore-missing-imports --scripts-are-modules --pretty .
+         mypy --no-strict-optional --ignore-missing-imports --scripts-are-modules --pretty .
```

Comment:
what does --no-strict-optional add for us?

Reply:
From https://github.com/pre-commit/mirrors-mypy

"By default, mypy will run with mypy --ignore-missing-imports, pre-commit runs mypy from an isolated virtualenv so it won't have access to those. To change the arguments, override the args as follows:
    hooks:
    -   id: mypy
        args: [--no-strict-optional, --ignore-missing-imports]
"